ehsm_kms_service/access_control_api.js

```diff
@@ -0,0 +1,236 @@
+const axios = require("axios")
+const logger = require("./logger")
+const { _result } = require("./common_function")
+const {
+  defense_apis,
+  DefenseCheckAccess,
+  DefenseReportTarget,
+  DefenseCheckNonce
+} = require("./access_control_server")
+
+const {
+  EHSM_CONFIG_DEFENSE_ENABLE,
+  EHSM_CONFIG_DEFENSE_SERVER_URL,
+} = process.env
+
+const _DefenseType = {
+  NONE: 1,
+  LOCAL: 2,
+  REMOTE: 3
+}
```

Comment:
does it need to distinguish the remote and lcoal?
not sure if the below works:
Disabled -- not set the URL
Enabled -- set a valid URL (a ocal host IP or remote IP)

so, the EHSM_CONFIG_DEFENSE_ENABLE should be deleted.

Reply:
![image](https://user-images.githubusercontent.com/29788476/159198870-0e01e9e7-ee24-40d8-ad17-1995214641a3.png)

Reply:
And also local mode is realized by calling function, a local host IP is invalid in docker